crates/re_renderer/shader/depth_cloud.wgsl

```diff
@@ -84,6 +84,17 @@ var<uniform> depth_cloud_info: DepthCloudInfo;
 @group(1) @binding(1)
 var depth_texture: texture_2d<f32>;
 
+struct DrawDataUniformBuffer {
+    size_boost_in_points: f32,
+    // In actuality there is way more padding than this since we align all our uniform buffers to
+    // 256bytes in order to allow them to be buffer-suballocations.
+    // However, wgpu doesn't know this at this point and therefore requires `DownlevelFlags::BUFFER_BINDINGS_NOT_16_BYTE_ALIGNED`
+    // if we wouldn't add padding here, which isn't available on WebGL.
+    _padding: Vec4,
+};
+@group(1) @binding(2)
+var<uniform> draw_data: DrawDataUniformBuffer;
```

Comment:
It doesn't make sense to have two separate uniform buffers on the same bind group. Why to through that pain? Either this is a uniform buffer that we put in a separate bind group that is then used for several depth clouds, or we meld it into the DepthCloudInfo.

Given that we don't expect massive amounts of depth clouds per draw data we can make the case of melding them here. *But* depth cloud draw data still has the concept on instances, so they _are_ batched like all other draw data. This means that per-drawdata should go to group 1 and per instance should go to group 2

Reply:
somewhat offtopic: Need for standardizing bind group usage is something that I noticed last week as well. We'll soon do some infrastructure about drawpass/drawphase and come up with rules then how the bindgroups 0-3 should be used accross all steps :)

Reply:
We need to have two separate UBs for the opaque phase and the outline phase, so conceptually at least it makes some sense to have a separate UB for "things that change between phases" 🤷

Reply:
fair enough but the existing UB doesn't get bigger by adding another element to it ;)

Reply:
let me know what you want me to do, if you think it is worth doing

Reply:
I'd like usage of group numbers to be consistent across renderers. The instance data using group `1` is the actual problem to begin with arguably.
But that said, let's leave it be, not worth spending extra human-cycles on this now.

Reply:
ok, you let me know in the comment below - I'll merge them